src/shared/a2dp-codecs.h

```diff
@@ -338,18 +340,28 @@
 #define LDAC_CHANNEL_MODE_DUAL          0x02
 #define LDAC_CHANNEL_MODE_STEREO        0x01
 
-#define LHDC_VENDOR_ID                  BT_COMPID_SAVITECH
-#define LHDC_CODEC_ID                   0x4C32
+#define LHDC_V2_VENDOR_ID               BT_COMPID_SAVITECH
+#define LHDC_V2_CODEC_ID                0x4C32
+
+#define LHDC_V3_VENDOR_ID               BT_COMPID_SAVITECH
+#define LHDC_V3_CODEC_ID                0x4C33
+
+#define LHDC_V5_VENDOR_ID               BT_COMPID_SAVITECH
+#define LHDC_V5_CODEC_ID                0x4C35
```

Comment:
I've got some question about these. There is v1, v2, v3 and v5. The v4 is non existent? Or you just reversed only these 4 variants? I'm asking because I'd like to pull the updated codec configurations as a first commit (before pulling the rest of it).

My Huawei phone reports 2 codec which very much resembles LHDC v4 and v6:
```
$ a2dpconf -x 7d020000344c03060150
<hex:7d020000344c03060150> {
  vendor-id:32 = 0x27d [HUAWEI Technologies Co., Ltd.]
  vendor-codec-id:16 = 0x4c34
  data:32 = hex:03060150
}
```

```
$ a2dpconf -x 7d020000364c03060150
<hex:7d020000364c03060150> {
  vendor-id:32 = 0x27d [HUAWEI Technologies Co., Ltd.]
  vendor-codec-id:16 = 0x4c36
  data:32 = hex:03060150
}
```

Reply:
V4 and LLAC are just flags for V3. 
> My Huawei phone reports 2 codec which very much resembles LHDC v4 and v6

![Screenshot from 2023-12-31 02-21-12](https://github.com/arkq/bluez-alsa/assets/48598263/87851a46-9bce-454a-bfa0-f81d641d24d8)

Reply:
Many thanks for explanation!